resources/benchmark.txt

```diff
@@ -6,72 +6,85 @@ DIJKSTRA forward with: 90397 nodes and 230142 edges
 saarland graph:
 amd64 Linux 4.4.0-97-generic 4 cpu(s)
 OpenJDK 64-Bit Server VM 25.131-b11
-Runtime arguments: -Dfile.encoding=UTF-8 -Xmx1g -XX:-OmitStackTraceInFastThrow -Dclojure.compile.path=/home/carocad/Proyectos/kamal/target/classes -Dkamal.version=0.1.4 -Dclojure.debug=false
+Runtime arguments: -Dfile.encoding=UTF-8 -Xmx1g -XX:-OmitStackTraceInFastThrow -Dclojure.compile.path=/home/carocad/Proyectos/kamal/target/classes -Dkamal.version=0.2.0 -Dclojure.debug=false
```

Comment:
Could you add a short guide to the wiki about benchmarks?

Reply:
can you tell me what would you expect to see there? I guess I am too deep in the topic so I think that most of the stuff should be clear but obviously that is never the case :rofl:

Reply:
Just basic stuff that allow people know which tools and commands they should use and how to update the files. A quick review of the purpose of the files would be very useful too.